docs/update-strategy/update-strategy.adoc

```diff
@@ -14,18 +14,34 @@ link:../../README.adoc[Return to Project Root]
   - <<inplace>>
     - <<example-using-the-inplace-strategy>>
     - <<recommendations-for-inplace-strategy>>
+    - <<inplace-ambient-mode-considerations>>
   - <<revisionbased>>
     - <<example-using-the-revisionbased-strategy>>
     - <<example-using-the-revisionbased-strategy-and-an-istiorevisiontag>>
+- <<updating-ambient-components>>
+  - <<updating-istiocni-ambient>>
+  - <<updating-ztunnel-ambient>>
+  - <<verifying-ambient-workloads>>
+  - <<updating-waypoint-proxies>>
+  - <<ambient-special-considerations>>
 
 [[update-strategy]]
 == Update Strategy
 
-The Sail Operator supports two update strategies to update the version of the Istio control plane: `InPlace` and `RevisionBased`. The default strategy is `InPlace`.
+The Sail Operator supports two update strategies: `InPlace` and `RevisionBased`. The default is `InPlace`.
+
+**For ambient mode updates:**
+
+IMPORTANT: **Only the InPlace strategy is supported for ambient mode.** RevisionBased updates are not supported because only one ztunnel instance can run in the cluster, and manual revision synchronization between Istio and ZTunnel CRDs would be required.
```

Comment:
I'm not sure what do you mean by that? It seems like the revisions would be possible if you manually sync CRDs?